client/components/admin_dashboard/Map.jsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+import { GoogleMap, LoadScript, Marker } from "@react-google-maps/api";
+import { googleApiKey } from "@/constants";
+import React, { useRef } from "react";
+
+export default function Map() {
+  const mapRef = useRef(null);
```

Comment:
Does this do anything? It's only assigned to, never read. Unless I'm not seeing it

Reply:
It's being referenced in the loader.
I was reading I should use ref to load the map since that part doesn't need to be re-rendered every time.

Reply:
Yeah but that's an assignment, nothing reads the value. It's like having an assigned but not read variable. Unless the library is doing some invisible magic to read from it